javascript/javascript1/week2/index4.js

```diff
@@ -0,0 +1,17 @@
+
+const clothes1="shorts and a t-shirt"
+const clothes2="jacket and pants"
+const clothes3="warm clothes"
+
+
+
+function weatherWear(num){
+    if(num>=0 && num<15)
+     return clothes1
+     else if (num>15)
+      return clothes2
+      else
+        return clothes3
+}
+
+console.log(weatherWear(24))
```

Comment:
The logic here is perfect! But check out your vars. Your function is suggesting shorts and a t-shirt for a temperature of 0 degrees ! 🥶🥶🥶